web-marketplace/src/pages/Roles/Roles.tsx

```diff
@@ -56,8 +58,8 @@ const Roles: React.FC<React.PropsWithChildren<unknown>> = () => {
   const initialValues: RolesFormSchemaType = useMemo(
     () => ({
       // In edit mode, use existing on chain project admin
-      // In creation mode, use current wallet address
-      admin: (isEdit ? onChainProject?.admin : wallet?.address) || '',
+      // In creation mode, use active account wallet address
+      admin: (isEdit ? onChainProject?.admin : activeAccount?.addr) || '',
```

Comment:
this is another bug in the Roles form I've encountered while fixing the initial one that happened as soon as you were logged in with an account with some wallet address (`activeAccount?.addr`) but the current account selected in Keplr was different (`wallet?.address`)